src/commands/preview.ts

```diff
@@ -123,6 +129,14 @@ export const configureForgeScriptInputs = ({ rpcUrl }: { rpcUrl: string }): stri
   if (!forgeArguments.includes('--broadcast')) forgeArguments.push('--broadcast');
   if (!forgeArguments.includes('--slow')) forgeArguments.push('--slow');
 
+  if (forgeArguments.includes('--verify') || forgeArguments.includes('--verifier-url'))
+    throw Error(
+      'Must verify contracts against the metal backend, please do not include --verify or --verifier-url flags',
+    );
+  forgeArguments.push(
+    `--verify --verifier-url ${METAL_SERVICE_URL}/contracts/verification/${previewId} --etherscan-api-key null`,
```

Comment:
is `--etherscan-api-key null` necessary

Reply:
kind of - when type of verifier is etherscan, yes